Apps/frontend/src/views/TimeSweepSlider.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+    import { timeSweep } from "../stores";
+    export let channel;
+</script>
+
+<div>
+    <input type="range" min="0" max="10" class="slider" bind:value={$timeSweep[channel]}>
```

Comment:
Maybe it could make sense moving min and max into the const file for easy configuration.